controllers/feeds/getFeedsV2.js

```diff
@@ -180,7 +181,8 @@ module.exports = async (req, res) => {
           } else {
             item.is_self =
               item.actor.id === req.userId || item.actor.id === myAnonymousUser?.user_id;
-            let newItem = await modifyAnonimityPost(item);
+            let newItem = await modifyFeedIsFollowingTarget(item, req.userId);
+            newItem = await modifyAnonimityPost(newItem);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of function calls here is important. The `modifyAnonimityPost` function modifies the `item` object and then it's passed to `modifyFeedIsFollowingTarget`. If `modifyAnonimityPost` changes any properties that `modifyFeedIsFollowingTarget` relies on, this could lead to unexpected behavior. Please ensure that the order of these function calls doesn't introduce bugs.
```diff
- let newItem = await modifyFeedIsFollowingTarget(item, req.userId);
- newItem = await modifyAnonimityPost(newItem);
+ let newItem = await modifyAnonimityPost(item);
+ newItem = await modifyFeedIsFollowingTarget(newItem, req.userId);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->